components/DescriptionList/README.md

```diff
@@ -0,0 +1,90 @@
+# Data list
```

Comment:
Didn't notice this before, but we should probably just refer to it as a description list everywhere. Now the docs are still calling it a data list. If we change this, we should leave a comment behind in FIgma about the naming so the Figma is updated as well.